src/app/app.module.ts

```diff
@@ -4,7 +4,7 @@ import {HttpModule} from '@angular/http';
 import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
```

Comment:
Add the `Roboto` font as a default font for all types of text (p, headings, etc).